README.md

```diff
@@ -102,9 +102,9 @@ hashi-ui can be controlled by both ENV or CLI flags as described below
 | `LOG_LEVEL` 	          | `log-level`               | `info`                  	| Log level to use while running the hashi-ui server - (`critical`, `error`, `warning`, `notice`, `info`, `debug`) |
 | `PROXY_ADDRESS`         | `proxy-address` 	      | `<empty>`               	| (optional) The base URL of the UI when running behind a reverse proxy (ie: example.com/nomad/)                   |
 | `LISTEN_ADDRESS`        | `listen-address`          | `0.0.0.0:3000`              | The IP + PORT to listen on |
-| `HTTPS_ENABLE`          | `https-enable`            | `false`                     | Use https protocol instead of http |
-| `SERVER_CERT`           | `server-cert`             | `<empty>`                   | Server certificate to use when https protocol is enabled |
-| `SERVER_KEY`            | `server-key`              | `<empty>`                   | Server key to use when https protocol is enabled |
+| `HTTPS_ENABLE`          | `https-enable`            | `false`                     | Use `--https-enable` or env `HTTPS_ENABLE=1` to enable HTTPS instead of HTTP for Hashi-UI |
```

Comment:
To be honest, I added it to make it consistent with NOMAD_ENABLE and CONSUL_ENABLE. Appending '=1' when used as an env var wasn't obvious to me but happy to remove